frontend/src/guidance/IgnoredCves.js

```diff
@@ -51,6 +100,7 @@ function IgnoredCves({ ignoredCves, setActiveCveHandler }) {
 export default withSuperAdmin(IgnoredCves)
 
 IgnoredCves.propTypes = {
-  ignoredCves: PropTypes.array.isRequired,
+  undetectedIgnoredCves: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+  detectedIgnoredCves: PropTypes.object.isRequired,
   setActiveCveHandler: PropTypes.func.isRequired,
```

Comment:
is it better to import the whole object or the individual prop types? if no performance/memory advantage this is fine

Reply:
I don't think there would be any performance loss either way. We already have these separated and the `undetectedIgnoredCves` are just strings since we aren't getting the vuln level from the api.